### DISCUSSION TITLE
Not using AWS

### DISCUSSION THREAD

**Discussion by vladiliushin** [apptension/saas-boilerplate#610] [Q&A]
Hello, 

I am really happy there is such great boilerplate available for the community. 

With that said, if I would to deploy it not on AWS infra, are there any pointers that you could give me? I see there are a lot of things connected with R53 as well as infra-as-a-code things, but if someone can point out intricacies of how not shoot one in the foot, that would be great!

Also, removing Stripe support should be as easy as creating/mocking subscription plans and removing djstripe module as well as modules from the front end?

Thanks and keep being awesome!

**Comment by mkleszcz** [comment]
At the moment SB is relying on AWS in three areas: IaC, CI/CD and services like S3 or RDS.
To deploy this boilerplate you wounl need to:
- remove IaC and use your own code or configure infrastructure manually
- remove dependencies from the application code that are using AWS services
- prepare your own CI/CD

About the second question:
Basically yes, we tried to keep the logic of the each module separated as much as possible so if there will be a need to remove or replace one with another, there will be an easy way to do this. Ofcourse, this is not as easy as removing the folder but it should help a lot.
